ui/app/AppLayouts/Wallet/views/walletconnect/sdk/src/index.js

```diff
@@ -14,21 +15,24 @@ window.wc = {
     statusObject: null,
 
     init: function (projectId) {
-        (async () => {
+        return new Promise(async (resolve, reject) => {
             if (!window.statusq) {
-                console.error('missing window.statusq! Forgot to execute "ui/StatusQ/src/StatusQ/Components/private/qwebchannel/helpers.js" first?');
-                return;
+                const errMsg = 'missing window.statusq! Forgot to execute "ui/StatusQ/src/StatusQ/Components/private/qwebchannel/helpers.js" first?'
+                console.error(errMsg);
```

Comment:
@stefandunca why didn't you use `wc.statusObject.bubbleConsoleMessage` here? This won't be logged in the app.

Reply:
The plan was to expose the console logs completely instead of adding an alternative way. However, it wasn't a priority. Also, the error message is propagated here through the promise reject, not lost.